custom_components/spaclient/sensor.py

```diff
@@ -1,25 +0,0 @@
-"""Support for Balboa Spa sensors."""
```

Comment:
I'm curious why you deleted this..  it kinda made sense to me, as in theory, we could have sensors for the filter cycle start times and be able to adjust them eventually, unless you have a better idea how to handle those?  I'd also be ok with them being a property of climate like you did with the time, assuming they are somehow adjustable..

Or should they be options?  Maybe.. This isn't a complaint, just kinda a thought of how to deal with them...

Reply:
The only reason I deleted it (for now) was because I moved the time component onto the climate rather than being a separate entity and there are no other items currently using it. So I didn't want it to get deployed if it never ends up being utilized in the future.

The more I looked at my hot tub entities in home assistant and perused other components, the less it made sense for the spa's current time to specifically be on it's own, especially with the logbook/history state change tracking. It just felt unnecessary. It's really only pertinent to know that the time on it is correct, but I don't need to have it tracked, per se, and don't think most users would find it useful as its own entity. If they do, it's easy to create a template on it as well, so allows for the ability without forcing it on people.

I do think keeping the time in sync (which was added as an option on the integration in this update), however, is extremely useful. So having the ability to still see the time for validity by looking at the state of the climate sensor, which is the main component I would say, made sense.

Agreed that the "sensor" could still have a future, but it's easy to bring back in once we've decided how to best utilize it. I'm still trying to work out how I want the filter information to be displayed and controlled since it has a start time, duration and on/off (at least for filter 2) component behind them.